packages/shortcuts-extension/src/components/ShortcutItem.tsx

```diff
@@ -460,6 +502,165 @@ export class ShortcutItem extends React.Component<
     );
   }
 
+  //navHandler = (event: React.KeyboardEvent): void => {}
+
+  // Handle key down for row navigation
+
+  handleRowKeyDown = (event: React.KeyboardEvent): void => {
+    const shortcutList = document.getElementById('Shortcuts-ShortcutList');
```

Comment:
This will not behave as intended if there is more than one shortcut list. I would strongly suggest rewriting the implementation so that the keydown event which should navigate is propagated up to `ShortcutList` component which would then perform focus switching.